### PR TITLE
fix: browser warnings

### PR DESCRIPTION
**Pull request by keita-determined**
## Description

Fix all browser warnings including UIKit page 

## Test Plan
- Check if there's no warnings in browser console, and validly fixed warnings

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/components/kit/LogViewer/LogViewer.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { Space } from 'antd';
 import React, { useCallback, useEffect, useLayoutEffect, useMemo, useRef, useState } from 'react';
-import { flushSync } from 'react-dom';
```
[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/components/kit/LogViewer/LogViewer.tsx:3]: yes, i introduced it due to this issue
it seems working without it now, but let me double-check.

https://user-images.githubusercontent.com/109113616/225971626-d5d167b0-e847-41ec-bb39-449fe05c76f4.mov

[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/components/kit/LogViewer/LogViewer.tsx]: @johnkim-det 
ok we still need `flushSync`, but it shows a warning in Design kit page due to `initialLogs`. Seems like `flushSync` itself isnt an issue, I think many `useEffect` in `LogViewer.tsx` are triggered inefficiently. For example, [this effect](https://github.com/determined-ai/determined/blob/master/webui/react/src/components/kit/LogViewer/LogViewer.tsx#L426) is triggered every render. If we refactor `LogViewer.tsx`, we can simplify code and the warning will be hopefully gone.
I just leave the warning for now.

File: webui/react/src/pages/DesignKit.tsx [hunk 2]
```diff
@@ -1092,6 +1092,9 @@ const PivotSection: React.FC = () => {
 };
 
 const PaginationSection: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [cunnrentPageSize, setCurrentPageSize] = useState<number>(1);
```
Comment (johnkim-det) [inline webui/react/src/pages/DesignKit.tsx:1096]: `cunnrentPageSize` typo

**Review by johnkim-det — APPROVED** [review]
looks good, thanks for addressing these

**Pull request merged**